### PR TITLE
Fixed error after new user login

### PR DESCRIPTION
**Pull request by prasanthchaduvula**
#### What
- Once a new user logs in after successful signup, the pages breaks while rendering

Screenshot 2024-02-22 at 9 56 17 AM

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -628,6 +628,7 @@ PLATFORMS
   arm64-darwin-22
   x86_64-darwin-21
   x86_64-darwin-22
+  x86_64-darwin-23
```
Comment (nisusam) [inline Gemfile.lock:631]: Is this relevant to this issue fix?

**Review by apoorv1316 — APPROVED** [review]
